core/domain/question_domain.py

```diff
@@ -96,11 +109,13 @@ class Question(object):
         question_data_schema_version: int. The schema version for the data.
         language_code: str. The ISO 639-1 code for the language this
             question is written in.
+        status: str. The status of the question among approved, rejected,
```

Comment:
Why is status being kept track of here? I thought this was going to be part of the review system (or the rights system if we're talking about private/public).

Reply:
I've removed all occurrences of states and attached functionalities.